src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -23,7 +27,11 @@ public static class MSBuildLocator
             "Microsoft.Build.Utilities.Core"
         };
 
+#if NET46
         private static ResolveEventHandler registeredHandler;
+#else
+        private static Func<AssemblyLoadContext, AssemblyName, Assembly> registeredHandler;
```

Comment:
>registeredHandler [](start = 73, length = 17)

nit: Pre-existing, but there's an inconsistency with resect to s_ prefix for static variables.

Reply:
It isn't. Removing.